packages/dotnet/ReadMe/HarJsonObjectModels/Root.cs

```diff
@@ -0,0 +1,12 @@
+﻿namespace ReadMe.HarJsonObjectModels
```

Comment:
These all new files?

Reply:
They are not! For some reason when I did a find and replace in VS Code, it also fixed all of the indentation on all of these files which made them different enough that when moved as well, it's just coming up as a completely new file. 

The old one is here: https://github.com/readmeio/metrics-sdks/blob/769f5b21abc7ffd06a7c96358080bd78094d5c59/packages/dotnet/Readme/HarJsonObjectModels/Root.cs